ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/results/tracing/flat/Action.java

```diff
@@ -109,6 +131,14 @@ public String getRefundAddress() {
     return refundAddress;
   }
 
+  public String getAuthor() {
```

Comment:
My understanding from working with @shemnon on JSON serialized code is that the team likes to have the right getters, but also to add `@JsonProperty` with the field names so it's obvious the method is used for JSON serialization.

Reply:
That is more of an emergent pattern than a deliberate one.  At one point getters and setters were prohibited from the codebase (pre public release).  That was changed but the `JsonProperty` annotations were not removed.

Reply:
it seem that no modification is necessary ?

Reply:
I'm OK with no modification.  There is no written style guide for this.